src/visualization/Turi Create Visualization/src/js/lib/elements/Plot/Plot.css

```diff
@@ -0,0 +1,19 @@
+.vega_container{
```

Comment:
You might want to use a vendor prefix (`tc_`) on any CSS we write; otherwise, it's possible something in Vega itself (or any of our other JS dependencies) could pick up and accidentally use our CSS. (Or, if it's intentional that we're using an element ID or class name from a dependent library, please add a comment in the CSS describing which library it's intended for.)